testsuite/features/build_validation/add_non_MU_repositories/alma9_minion_build_clm.feature

```diff
@@ -44,3 +45,35 @@ Feature: Add the Alma 9 distribution custom repositories
     And I enter "Initial build" as "message"
     And I click the environment build button
     Then I should see a "Version 1: Initial build" text
+
+@uyuni
+  Scenario: Create a CLM project to remove AppStream metadata from Alma 9
+    When I follow the left menu "Content Lifecycle > Projects"
+    And I follow "Create Project"
+    And I enter "Remove AppStream metadata from Alma 9" as "name"
+    And I enter "no-appstream-alma-9" as "label"
+    And I click on "Create"
+    Then I should see a "Content Lifecycle Project - Remove AppStream metadata from Alma 9" text
+    When I click on "Attach/Detach Sources"
+    And I wait until I do not see "Loading" text
+    And I select "AlmaLinux 9 (x86_64)" from "selectedBaseChannel"
+    And I check "Uyuni Client Tools for AlmaLinux 9 (x86_64) (Development)"
```

Comment:
The channel for the Uyuni devel client tools are named that way. That is nothing I introduced. See
<img width="1146" alt="image" src="https://github.com/uyuni-project/uyuni/assets/12104291/99bbf20e-5dfe-406a-bf80-9eed30f7ecb5">
<img width="1166" alt="image" src="https://github.com/uyuni-project/uyuni/assets/12104291/aaa7fdd5-73c9-4539-a9dc-1dc94bd871da">

Reply:
We agreed on using `Uyuni:Master` instead of `Uyuni:Stable`. Because of that, our client tools channel is named that way.

Reply:
Yes, what I mean is that if there is only one entry "Uyuni Client Tools for AlmaLinux 9 (x86_64) (Development)" and there is no entry "Uyuni Client Tools for AlmaLinux 9 (x86_64)" for now. I think we better use "Uyuni Client Tools for AlmaLinux 9 (x86_64)" on this step, as this will select the same entry with or without this word appended, and once SCC change the sub-product name to "Uyuni Client Tools for AlmaLinux 9 (x86_64)" we will not need to update our code to adapt to it.

And this is something that happens very often, we have Beta channels coming in and out depending the phase of our product.